test/mp4-inspector.test.js

```diff
@@ -403,7 +404,7 @@ QUnit.test('can parse a moov', function(assert) {
           flags: new Uint8Array([0, 0, 0]),
           edits: [{
             segmentDuration: 0,
-            mediaTime: 1152921504606847000,
+            mediaTime: BigInt('0x1000000000000000'),
```

Comment:
This number was 100% wrong before and relying on consistent incorrect rounding in javascript.